### ISSUE TITLE
allows users to retract a vote

### ISSUE THREAD

**Issue by samdec11** [stride-nyc/remote_retro#275]
As a participant,
I want to be able to detract a vote from an item I've accidentally voted on
So that I can allocate my votes where I actually want them.

__By Way Of Explanation:__ in the current voting experience on production, we have a 'Vote' button, which users sometimes click multiple times thinking that it will toggle/retract the vote they've just submitted. Instead, it logs another vote for the same idea. In other cases, users simply want to re-allocate their votes upon rethinking their choices. We have developed a new interface, and need to enable this retraction behavior.

__Old Interface:__
![image](https://user-images.githubusercontent.com/4386645/52955769-2b1af580-3353-11e9-851e-d5b705066449.png)

__New Interface:__ (behind a feature flag)
![image](https://user-images.githubusercontent.com/4386645/52955806-44bc3d00-3353-11e9-8382-b401f9badab7.png)

__Acceptance Criteria:__

- [ ] when in the new voting experience, clicking an enabled 'minus' button results in a vote being removed across all connected clients
  - this removal does _not_ have to implement:
    - optimistic UI updates in the detracting user;s UI. that will be covered in a follow-up story.
    - a decrement animation. this will also be covered in a follow-up story

__Required Dev Note:__ to develop in the new experience, you must call `localStorage.subtractVoteDev` in the JavaScript console.

__Helpful Links:__
  - [`VotingInterfaceNew`](https://github.com/stride-nyc/remote_retro/blob/6efb4c6deb2bd5252e0941910dd1f9407f6d75e3/web/static/js/components/voting_interface_new.jsx)
    - component in new voting experience
  - [votes.js](https://github.com/stride-nyc/remote_retro/blob/6efb4c6deb2bd5252e0941910dd1f9407f6d75e3/web/static/js/redux/votes.js)
    - component in new voting experience
  - [Ecto.Repo 2.x Docs](https://hexdocs.pm/ecto/2.0.4/Ecto.Repo.html)

**Comment by vanderhoop** [comment]
@samdec11 just going through the backlog, and wanted to update you on my thinking with this one. Once the grouping stage is complete, we'll have users vote on groups of ideas, rather than on individual ideas. we'll revisit this then!

**Comment by vanderhoop** [comment]
this feature is now live. Thanks to all involved in the effort, including @grantziolkowski @JennMiller @rakavai and @edem-ca and @samdec11!
